subclass/GM Binder; Water Domain.json

```diff
@@ -0,0 +1,204 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "waterdomain",
+				"abbreviation": "C:WD",
+				"full": "Divine Domain: Water",
+				"url": "https://www.gmbinder.com/share/-LIdfvCAVzWd43w7WtT4",
+				"authors": [
+					"GM Binder",
+					"Mathias Kollros"
+				],
+				"convertedBy": [
+					"calebmeyer"
+				],
+				"version": "3.2"
+			}
+		],
+		"dateAdded": 1234,
+		"dateLastModified": 1234,
+		"_dateLastModifiedHash": "todo"
```

Comment:
I wasn't sure how the meta updater works, so I put some garbage values here. I can remove these lines if needed.

Reply:
you can remove the `_dateLastModifiedHash`; it's auto-generated when I index things.
As for the other two, you can use e.g. https://www.epochconverter.com/ and grab the timestamp at the top